test/jdk/java/security/Security/ClassLoaderDeadlock/ClassLoaderDeadlock.java

```diff
@@ -21,7 +21,15 @@
  * questions.
  */
 
-// See bug 5094825 / ClassLoadDeadlock.sh
+/*
+* @test
+* @bug 5094825
+* @summary verify no deadlock if crypto provider in other classloader is used to verify signed jars
+* @modules java.base/java.security
+* @library ./Deadlock.jar
+* @compile -g provider/HashProvider.java
+* @run main/othervm/timeout=30 -Djava.awt.headless=true ClassLoaderDeadlock
```

Comment:
The `-Djava.awt.headless=true` is likely not needed but since it was used in the original test script, I think it's OK to carry it over.